webdriver_wharf/app.py

```diff
@@ -15,13 +15,24 @@
 
 pool = set()
 logger = logging.getLogger(__name__)
+
+# Environment variables
+#
+# Name of the docker image to use
 image_name = os.environ.get('WEBDRIVER_WHARF_IMAGE', 'cfmeqe/sel_ff_chrome')
+# Set to True if the container runs a VNC service
+vnc_enabled = os.environ.get('WEBDRIVER_WHARF_IMAGE_VNC_ENABLED', True)
+# Set to True if the container runs an httpd service
+httpd_enabled = os.environ.get('WEBDRIVER_WHARF_IMAGE_HTTPD_ENABLED', True)
```

Comment:
For all occurances: http, not httpd. HTTP is the generic service name, httpd is apache's web server.